pyogrio/tests/test_geopandas_io.py

```diff
@@ -186,6 +186,79 @@ def test_read_fids_force_2d(test_fgdb_vsi):
         assert not df.iloc[0].geometry.has_z
 
 
+#@pytest.mark.filterwarnings("ignore: Layer")
+def test_read_sql(naturalearth_lowres):
+    # empty filter should return full set of records
```

Comment:
This can be removed since it is already covered by other tests, given default of `sql=None`.